persistence-model/src/main/java/org/oxauth/persistence/model/ClientAttributes.java

```diff
@@ -55,7 +55,20 @@ public class ClientAttributes implements Serializable {
     private List<String> introspectionScripts;
 
     @JsonProperty("rptClaimsScripts")
-    private List<String> rptClaimsScripts;
+    private List<String> rptClaimsScripts;    
+
+    @JsonProperty("redirectRegex")
+    private String redirectRegex;
+    
+	@JsonProperty("umaRPTClaimsScripts")
+	private List<String> umaRPTModificationScripts;
+	
+	@JsonProperty("updateTokenScripts")
+	private List<String> updateTokenScripts;
+	
+	@JsonProperty("defaultPromptLogin")
+	private String defaultPromptLogin;
```

Comment:
These 4 attributes are not supported in oxauth. They are supported in jans only. What is the sense of adding it ? We shoudn't do it.

Reply:
@yuriyz  Mike suggested to add these fields.

Reply:
To oxauth ? It must be some mis-understanding. cc @nynymike

Reply:
Yuriy is right... if they are not in oxAuth they are not needed.

Reply:
Ok so I m closing this PR.